charts/sysdig-deploy/values.yaml

```diff
@@ -24,6 +24,14 @@ agent:
 
 nodeAnalyzer:
   enabled: true
+  nodeAnalyzer:
+    kspmAnalyzer:
+      image:
+        tag: 1.17.8-rc
+
+    kspmCollector:
+      image:
+        tag: 1.17.7-rc
```

Comment:
@galshnapp-sysdig - what is the rationale for putting these values here? I'm curious because none of the other components list there image tags at this level since the `values.yaml` file for `sysdig-deploy` is mostly just for toggling the subcharts on/off. Pushing these down into the `node-analyzer` chart's `values.yaml` file would also reduce the complexity of this PR as you wouldn't have to bump the `sysdig-deploy` version number either since our GitHub actions will take care of that automatically.

Reply:
Hey, I Just used this to run tests on this branch.
Removed now.